DataFormats/Headers/include/Headers/DataHeader.h

```diff
@@ -628,6 +628,7 @@ struct DataHeader : public BaseHeader {
   using SplitPayloadIndexType = uint32_t;
   using SplitPayloadPartsType = uint32_t;
   using PayloadSizeType = uint64_t;
+  using GlobalIDType = uint64_t;
 
   //static data for this header type/version
   static constexpr uint32_t sVersion{1};
```

Comment:
sVersion should probably be bumped.

Reply:
indeed, in the past we got away without bumping this, it's a good moment to start:)